src/test/java/com/scalyr/integrations/kafka/AddEventsClientTest.java

```diff
@@ -438,6 +447,50 @@ public void testTooLargeAddEventsSkipped() throws Exception {
   }
 
 
+  /**
+   * Verify Add Events Requests that exceed maximum add events payload size before compression are not sent.
+   *
+   * In this scenario we test a request which is larger than 6 MB before compression, but smaller than 6 MB
+   * after compression.
+   */
+  @Test
+  public void testTooLargeAddEventsSkippedDeflateCompressedRequest() throws Exception {
+    final int numEvents = 6;
+    final byte[] largeMsgBytes = new byte[1000000];
+    Arrays.fill(largeMsgBytes, (byte)'a');
+    final String largeMsg = new String(largeMsgBytes);
+    AddEventsResponse addEventsResponse;
```

Comment:
Same as above.  `addEventsResponse` assignment is not needed since we don't do anything with the assigned variable.

Reply:
Removed in 36962e6503704c1aa5f95f869f5bcf77704dc66f (old habits die hard :D)